packages/text-annotator/src/api/scrollIntoView.ts

```diff
@@ -16,59 +16,68 @@ const getScrollParent = (el: Element) => {
     return getScrollParent(el.parentElement);
 };
 
-export const scrollIntoView = (
-  container: HTMLElement, store: TextAnnotationStore
-) => (annotationOrId: string | TextAnnotation) => {
-  const id = 
-    typeof annotationOrId === 'string' ? annotationOrId : annotationOrId.id;
+// Executes scroll on an annotation with a valid DOM range selector
+const scroll = <I extends TextAnnotation = TextAnnotation>(
+  store: TextAnnotationStore<I>,
+  target: I['target'],
+  scrollParent: Element
+) => {
+  // Parent bounds and client (= visible) height
+  const parentBounds = scrollParent.getBoundingClientRect();
+  const parentHeight = scrollParent.clientHeight;
+  const parentWidth = scrollParent.clientWidth;
+
+  // Position of the annotation relative to viewport
+  const annotationBounds = target.selector[0].range.getBoundingClientRect();
 
-  // Executes scroll on an annotation with a valid DOM range selector
-  const scroll = (target: TextAnnotationTarget) => {
-    // Parent bounds and client (= visible) height
-    const parentBounds = scrollParent.getBoundingClientRect();
-    const parentHeight = scrollParent.clientHeight;
-    const parentWidth = scrollParent.clientWidth;
+  /*
+   Note: `getBoundingClientRect` seems to return the wrong height!
+   (Includes block elements?)
+   Instead, the normalized height is used from the spatial index!
+  */
+  const { width, height } = store.getAnnotationBounds(target.annotation);
 
-    // Position of the annotation relative to viewport
-    // Note: first selector is not necessarily top one...
```

Comment:
This line seems contradictory to the following one 😅 https://github.com/recogito/text-annotator-js/blob/c07bdaf5a91b3e804c8fb2f4bb0f07b8481f6fd1/packages/text-annotator/src/api/scrollIntoView.ts#L61-L62